schema/venom.schema.json

```
@@ -19,7 +19,8 @@
         "testcases": {
           "type": "array",
           "items": { "$ref": "#/definitions/test_case" }
-        }
+        },
+        "describe": { "type": "string" }
```

```suggestion
        "description": { "type": "string" }
```

`describe` doesn't seem to exist on Venom: https://github.com/ovh/venom/blob/master/types.go#L116

---

Fixed